### PR TITLE
Add EA Desktop support

### PR DESCRIPTION
**Pull request by ashemedai**
Add EA Desktop IDs for Dragon Age: Origins and Dragon Age 2.

### CODE REVIEW

**Comment by ashemedai** [comment]
This is the result from a custom `EA Games` installation directory:
image

The Dragon Age entries have two entries as some entries at EA are legacy branded and use that id in their installation (multiple content ids mapped to the same offer id).

File: eadesktop_utils.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    Find the list of Origin games installed.
+
+    Returns:
+        A mapping from Origin manifest IDs to install locations for available
+        Origin games.
+    """
+    games: Dict[str, Path] = {}
+
+    local_app_data_path = os.path.expandvars("%LocalAppData%")
+    ea_desktop_settings_path = Path(local_app_data_path).joinpath("Electronic Arts", "EA Desktop")
```
Comment (ashemedai) [inline eadesktop_utils.py:21]: TODO: Add short-circuit code if this does not exist.

**Comment by ZashIn** [comment]
Can you add `GameEaDesktopId` and where to find them to the `README.md` (under [#List of valid keys](..#list-of-valid-keys) and [#Extra features](..#extra-features)), similar to the other launcher IDs?

**Comment by ashemedai** [comment]
@ZashIn Will do that later today.

**Comment by ashemedai** [comment]
@ZashIn Sorry, life happened. Added now.

**Comment by Holt59** [comment]
Merged into `qt6`, will probably need some testing.

**Pull request closed without merging**